lte/gateway/c/core/oai/tasks/sgw/sgw_context_manager.cpp

```diff
@@ -157,35 +157,16 @@ sgw_cm_create_bearer_context_information_in_collection(teid_t teid) {
     return NULL;
   }
 
-  OAILOG_DEBUG(
-      LOG_SPGW_APP,
-      "sgw_cm_create_bearer_context_information_in_collection " TEID_FMT "\n",
-      teid);
-
-  bstring b = bfromcstr("pgw_eps_bearer_ctxt_info_apns");
-  new_bearer_context_information->pgw_eps_bearer_context_information.apns =
-      obj_hashtable_ts_create(32, NULL, NULL,
-                              (void (*)(void**))pgw_lite_cm_free_apn, b);
-  bdestroy_wrapper(&b);
-
-  if (new_bearer_context_information->pgw_eps_bearer_context_information.apns ==
```

Comment:
Where is APN collection getting created now?

Reply:
Hi @pruthvihebbani, this removed code was not used so I cleaned it up. The apn information is available within pdn session, https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/lib/3gpp/3gpp_23.401.h#L122